src/web/components/ParticipantManagement/ParticipantRequestForm.stories.tsx

```diff
@@ -63,6 +63,12 @@ ParticipantApproval.args = {
     { id: 2, typeName: 'Type 2' },
```

Comment:
While we are here, can we please change the name of the participant so that the site dropdown starts off with a match to begin with? That way the developer does not need to check what sites are being provided in the story.
![image](https://github.com/IABTechLab/uid2-self-serve-portal/assets/137864392/4024aab5-99dd-446f-9ed2-4b2a26ce245a)

Perhaps `Test Participant`?
![image](https://github.com/IABTechLab/uid2-self-serve-portal/assets/137864392/3872a12c-0d6a-4357-ba28-3e5a48c44b5b)

Reply:
Perhaps this should be an additional story, rather than changing the existing one.

Reply:
Yeah I added a second so the search can still be tested